docs/material-top-tab-navigator.md

```diff
@@ -317,7 +317,7 @@ An object containing the props for the tab bar component. It can contain the fol
 - `scrollEnabled` - Whether to enable scrollable tabs.
 - `tabStyle` - Style object for the tab.
 - `indicatorStyle` - Style object for the tab indicator (line at the bottom of the tab).
-- `labelStyle` - Style object for the tab label.
+- `labelStyle` - Style object for the tab label (specifying a color here will override the color specified in `activeTintColor` and `inactiveTintColor` for the label).
```

Comment:
nit: could be a normal sentence instead of bracket

Reply:
@satya164 ok now?